src/components/TodoItem/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoItem';
```

Comment:
The code looks good and follows the specified rule about exporting. There are no issues found in this file. 😊